### PR TITLE
Update pallai.txt

### PR DESCRIPTION
**Pull request by Vicky7873**
for conflict

### CODE REVIEW

**Review by Vicky7873 — COMMENTED** [review]
please check

File: pallai.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@ lets try the merge conflict
 with merge conflict
 
 ========================
-New code from dev 3
+New code for conflict
```
Comment (bhikiLead) [inline pallai.txt:8]: please tell me why you did this

**Review by bhikiLead — CHANGES_REQUESTED** [review]
please change it to qa

**Review by Vicky7873 — COMMENTED** [review]
changes has been done

File: pallai.txt (hunk 1 above)
Comment (Vicky7873) [reply pallai.txt]: please check the chnages

**Review by bhikiLead — APPROVED** [review]
approved please merge

**Comment by Vicky7873** [comment]
close this 2nd pr

**Pull request merged**